packages/storybook/stories/Accessibility.stories.mdx

```diff
@@ -1,6 +1,10 @@
 import { Meta } from '@storybook/addon-docs';
 
-<Meta id="accessibility" title="Concepts/Accessibility" />
+<Meta
+  id="accessibility"
+  title="Concepts/Accessibility"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
```

Comment:
nit: this is used in every pure docs file, can this be moved to a constant and imported in case it needs changed in the future
```suggestion
import { docsOnlyTabs } from '../constants';
...
  parameters={docsOnlyTabs}
```

Reply:
great suggestion will get on that!

Reply:
https://github.com/storybookjs/storybook/pull/11752/files

![image](https://user-images.githubusercontent.com/79329532/144193674-a3bc9df2-676b-4db8-979e-50506fb6ef17.png)

It looks like we need to manually pass in parameters to our stories.mdx files